Fix CSV UTF8 Bug

cgaber2045 opened the pull request.

cgaber2045 committed:
fix(utf8): Bug should be fixed, but also updated message on bulk enrollment.

cgaber2045 committed:
fix(utf8): Fixed bug

ztraboo commented on lms/djangoapps/instructor/settings/common.py:
```diff
@@ -69,7 +69,7 @@ def plugin_settings(settings):
         # .. toggle_creation_date: 2014-10-21
         # .. toggle_tickets: https://github.com/edx/edx-platform/pull/5670
-        'ALLOW_AUTOMATED_SIGNUPS': False,
+        'ALLOW_AUTOMATED_SIGNUPS': True,
```
@cgaber2045 I'd prefer that we not override the configuration like this but rather in the `lms/env/private.py` file like so. That way the platform keeps its default and we can keep our configuration for EducateWorkforce separate. I'll go ahead and back out this change.

```
# edx-platform/lms/envs/private.py

#################### Instructor Plugin Settings #######################
# edx-platform/lms/djangoapps/instructor/settings/common.py
FEATURES.update({

    # Membership > CSV Enrollment
    'ALLOW_AUTOMATED_SIGNUPS': True,

})
```

cc: @becdavid

ztraboo committed:
fix: Setting automated signups to default `maple` release.

Need to set this in `lms/envs/private.py` for configuration overrides rather than changing the default outlined by the `maple` release.
https://github.com/CUCWD/edx-platform/pull/157/files#r841719995
